securedrop/encryption.py

```diff
@@ -155,16 +170,16 @@ def decrypt_journalist_reply(self, for_source_user: "SourceUser", ciphertext_in:
                 secret_key=for_source.pgp_secret_key,
                 passphrase=for_source_user.gpg_secret,
             ).decode()
-        # TODO: Migrate gpg-stored secret keys to database storage for Sequoia
+        # In practice this should be uncreachable unless the Sequoia secret key migration failed
```

Comment:
Error logging is usually disabled for the SI, but in cases where it's not (testing, troubleshooting) should we emit an error here to alert admins that there was a failed key migration earlier?